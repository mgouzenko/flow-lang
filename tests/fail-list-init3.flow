// Should not be able to init list of type that does not match declaration

int main() {
  list <int> test = [5.2, 5.0]; // FAIL
}