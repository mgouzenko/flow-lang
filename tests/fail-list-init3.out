Fatal error: exception Failure("test: Declaration type does not match expression")